bitnami/kube-prometheus/README.md

```diff
@@ -258,6 +258,7 @@ The following table lists the configurable parameters of the kube-prometheus cha
 | `prometheus.thanos.image.tag`                              | Thanos image tag                                                                                        | `{TAG_NAME}`                                                                                                                            |
 | `prometheus.thanos.image.pullPolicy`                       | Thanos image pull policy                                                                                | `IfNotPresent`                                                                                                                          |
 | `prometheus.thanos.image.pullSecrets`                      | Specify docker-registry secret names as an array                                                        | `[]` (does not add image pull secrets to deployed pods)                                                                                 |
+| `prometheus.thanos.prometheusUrlOverride`                  | Specify custom prometheus url                                                                           | `""`                                                                                                                                    |  
```

Comment:
I think the parameter should be called `prometheusUrl` alone. No need for the `Override` part.